src/status_im2/subs/communities.cljs

```diff
@@ -150,6 +151,19 @@
     (get communities identity)
     counts)))
 
+(re-frame/reg-sub
+ :communities/my-pending-requests-to-join
+ (fn [[]]
+   [(re-frame/subscribe [:my-pending-requests-to-join])])
+ (fn [[requests]]
+   requests))
+
+(re-frame/reg-sub
+ :communities/my-pending-request-to-join
+ :<- [:communities/my-pending-requests-to-join]
+ (fn [requests [_ community-id]]
+   (get requests (name community-id))))
```

Comment:
Curious, isn't the `community-id` a string already? Why the need to convert it to a string? Could it be a keyword? is that why `name` is used?

Reply:
it's a string already, I kind of assumed we would want to make it a keyword. But considering it now, I should probably just leave it as a string. Or as we discussed above use oops to handle it early as possible.